### PR TITLE
[17.0][MIG] partner_company_default

### PR DESCRIPTION
**Pull request by giarve**
Improvement over #1759  as it has [all fixes](https://github.com/OCA/partner-contact/pull/1712)

### CODE REVIEW

**Comment by rousseldenis** [comment]
@giarve Thanks for this. But you need to take the whole commit history following the migration procedure: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0#technical-method-to-migrate-a-module-from-160-to-170-branch

**Comment by giarve** [comment]
> @giarve Thanks for this. But you need to take the whole commit history following the migration procedure: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0#technical-method-to-migrate-a-module-from-160-to-170-branch

Thanks for pointing out. I shallow cloned, and the commit was not included in the rebase. Fixed.

File: partner_company_default/readme/DESCRIPTION.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+## Background
+
+Up to 12.0, Odoo used to assign a company by default when partner was
+added, however the default company proposal has been removed since 13.0
+with multi-company refactoring. This causes inconveniences to users who
+want to seggregate partner records between companies.
+
+ref.
+<https://github.com/odoo/odoo/commit/25714692b23624d04a43862db26ebcf802b89399>
```
Comment (yostashiro) [inline partner_company_default/readme/DESCRIPTION.md]: This could be moved to CONTEXT.md, if that suits better.

**Review by yostashiro — APPROVED** [review]
LGTM. Thanks for the migration!

File: partner_company_default/readme/DESCRIPTION.md (hunk 1 above)
Comment (giarve) [reply partner_company_default/readme/DESCRIPTION.md]: Done. Thanks!

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**